Trigger/fcl/templates.fcl

```diff
@@ -204,14 +204,21 @@ Trigger : {
     outputs: {
 	triggerOutput : { 
 	    module_type: RootOutput
-	    fileName: "filteredOutput.root"
+	    fileName: "filteredOutput_%#.root"
```

Comment:
Is the %# replaced with something meaningful in the python code?  If the file isn't valid fhicl we should name it something else: .fclpy?

Reply:
to make it short and concise, I started learning how in OTS we will be assigning the subRun number (it will come from the CFO) and how to create an art file per subRun. Kyle pointed me this page: https://cdcvs.fnal.gov/redmine/projects/art_root_io/wiki/Output-file_handling 
so the way I set the RooutOutput, should create a new .art file every 1000 events. The %# should be increased automatically as the files get created.